README.md

```diff
@@ -73,6 +73,16 @@ jobs:
           fail: true
 ```
 
+You may want to add additional arguments to the above.  In particular if you're testing a site directly from the file-system (as in the above), you'll likely want to set the argument `--base .` to ensure that **all links** in the files are tested.  You don't need to do this if you're testing a hosted site.
```

Comment:
I understand the context because I come from [this issue](https://github.com/lycheeverse/lychee/issues/1265), but I think it’s not clear enough as proposed. Small suggestion:

```suggestion
You may want to add additional arguments to the above. In particular if you're testing a site directly from the file-system (as in the above), you'll likely want to set the argument `--base .` to ensure that **all links** (including relative paths) in the files are tested. You don't need to do this if you're testing a hosted site.
```

Reply:
Hi @meduzen - to an extent I've lost track of this as I haven't touched that setup in a bit.  
But IIRC it was specifically about *root*-relative paths, not just relative ones.  So those that begin with `/some-path/file.md` (emphasis on the first `/`).  Do you agree?

Reply:
Yes! It’s at least [my own use case](https://github.com/meduzen/blog/blob/fb90a85ee7bb9562b58bb7c04b7673a0a1baf674/.github/workflows/links.yml#L20) where I have `args: './vitepress/**/*.md --base ./vitepress/public'` because my _root-relative_ paths (e.g. `src="/pictures/pic.jpg"`) are files located in the folder mentioned by `--base`: so the “lychee” path on the filesystem is `./vitepress/public/pictures/pic.jpg`.